core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -10492,7 +10492,7 @@ private void checkLoptOptimizeJoinRule(LoptOptimizeJoinRule rule) {
   /** Test case of
    * <a href="https://issues.apache.org/jira/browse/CALCITE-6973">[CALCITE-6973]
    * Add rule for convert Minus to Filter</a>. */
-  @Test void testMinusToFilterRule() {
+  @Test void testMinusToFilterRuleWithTwoFilter() {
```

Comment:
```suggestion
  @Test void testMinusToFilterRuleWithTwoFilters() {
```

Reply:
Done!

Reply:
The local test passes, but pr-head always fails. I will push the code later. Sorry.

Reply:
Thanks, I saw this error. But it can indeed pass through locally. I agree with your idea that we cannot merge the PR of CI. Give me some time, and I will find a way to get CI approved.

Reply:
It's very weird that it passes locally and fails in CI, it would help tracking when this started to help figure out the source of the issue.